third_party/manifest-parser.js

```diff
@@ -157,7 +157,7 @@ function parseStartUrl(jsonInput, manifestUrl, documentUrl) {
 }
 
 function parseDisplay(jsonInput) {
-  let display = parseString(jsonInput.display, true);
+  const display = parseString(jsonInput.display, true);
```

Comment:
@juliantoledo Did you use https://github.com/GoogleChrome/gulliver/blob/master/third_party/install.sh to generate this? Just wondering why the updated `Color.js` isn't here as well. (Which seems to include some awkward changes, but still probably better to commit both together.)